<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>70</y>
      <width>301</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Par√¢metros</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>261</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QSlider" name="HSraioX">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLCDNumber" name="lcdNumber_2"/>
      </item>
      <item row="6" column="1">
       <widget class="QSlider" name="HSraioZ">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Elipse</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSlider" name="HSraioY">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLCDNumber" name="lcdNumber_4"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Rz</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLCDNumber" name="lcdNumber_3"/>
      </item>
      <item row="5" column="1">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Ry</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Rx</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>261</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QSlider" name="HSraioEsfera">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLCDNumber" name="lcdNumber"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Esfera</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Raio</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>261</width>
       <height>128</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QSlider" name="HSX">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLCDNumber" name="lcdNumber_5"/>
      </item>
      <item row="5" column="1">
       <widget class="QSlider" name="HSZ">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="HSY">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>DimX</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLCDNumber" name="lcdNumber_6"/>
      </item>
      <item row="3" column="2">
       <widget class="QLCDNumber" name="lcdNumber_7"/>
      </item>
      <item row="4" column="1">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>DimY</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>DimZ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Caixa</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>261</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Cor</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="HSvermelho">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>RED</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="HSverde">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLCDNumber" name="lcdNumber_10"/>
      </item>
      <item row="1" column="2">
       <widget class="QLCDNumber" name="lcdNumber_8"/>
      </item>
      <item row="5" column="2">
       <widget class="QLCDNumber" name="lcdNumber_13"/>
      </item>
      <item row="5" column="1">
       <widget class="QSlider" name="HSopacidade">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLCDNumber" name="lcdNumber_12"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_17">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>OPACIDADE</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_14">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>BLUE</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>GREEN</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLCDNumber" name="lcdNumber_9"/>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <family>MS Serif</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>BLUE</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSlider" name="HSazul">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>380</y>
       <width>96</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>851</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Visualiza√ß√µes dos planos</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>811</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <family>MS Serif</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Deslizar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="HSdeslizar">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_11"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="buttonPXY">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Plano XY</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonPXZ">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Plano XZ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonPYZ">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PlanoYZ</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1161</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="Line" name="line_10">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="putVoxel">
       <property name="text">
        <string>Colocar Voxel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cutVoxel">
       <property name="text">
        <string>Retirar Voxel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="putBox">
       <property name="text">
        <string>Colocar Caixa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cutBox">
       <property name="text">
        <string>Retirar Caixa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="putSphere">
       <property name="text">
        <string>Colocar Esfera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cutSphere">
       <property name="text">
        <string>Retirar Esfera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="putEllipse">
       <property name="text">
        <string>Colocar Elipse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cutEllipse">
       <property name="text">
        <string>Retirar Elipse</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>851</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QWidget" name="widget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1189</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>HSdeslizar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_11</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>769</x>
     <y>583</y>
    </hint>
    <hint type="destinationlabel">
     <x>839</x>
     <y>591</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HSazul</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_9</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1089</x>
     <y>611</y>
    </hint>
    <hint type="destinationlabel">
     <x>1135</x>
     <y>615</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HSverde</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_10</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1097</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>1122</x>
     <y>573</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HSvermelho</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_8</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1092</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>1112</x>
     <y>532</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HSZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_6</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1093</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>1118</x>
     <y>469</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HSY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_7</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1089</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>1124</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HSX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1084</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>1113</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HSraioZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_4</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1082</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>1115</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HSraioY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1096</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>1111</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HSraioX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1086</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>1111</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HSraioEsfera</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1086</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>1123</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HSopacidade</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_13</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1071</x>
     <y>635</y>
    </hint>
    <hint type="destinationlabel">
     <x>1135</x>
     <y>639</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
